docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '2'
+services:
+  frontend:
+    build: frontend
+    image: lb-frontend
```

Comment:
for the image names can we expand out `lb` to `liberty-bikes` or `libertybikes`?